requirements.txt

```diff
@@ -29,127 +29,123 @@
 -e git+https://github.com/zenodo/zenodo-migrator.git#egg=zenodo-migrator
```

Comment:
When we deploy we have to take care of removing old dependencies. In particular Flask-Security-Fork and many test dependencies.